api/test/wfl/system/automation_test.clj

```diff
@@ -66,8 +66,8 @@
     (fixtures/with-fixtures
       [(util/bracket clone-workspace delete-workspace)]
       (fn [[workspace]]
-        (let [workload (endpoints/create-workload
-                        (covid-workload-request workspace))]
+        (let [workload (-> (covid-workload-request workspace)
+                           (endpoints/create-workload))]
```

Comment:
This is fine, of course, but a linter complains when -> has fewer than 3 forms.
Suggest expanding it or moving workspace out:
```
(endpoints/create-workload
  (covid-workload-request workspace))
```
or
```
(-> workspace
  covid-workload-request
  endpoints/create-workload)
```
… or the next lint victim can address it.

Reply:
TIL!  I'll just switch back.